### DISCUSSION TITLE
DefaultOption for multiselect?

### DISCUSSION THREAD

**Discussion by noque-lind** [MobileReality/react-native-select-pro#225] [Q&A]
Hello,

Is there anyway to specify multiple default selected values? Currently it seems it is only possible to specify one defaultOption that can be shown initially. However in my app, I use the component in multiple={true} mode, but would need to fill in the component with the previously selected options.

**Comment by RuchkoVlad** [comment]
I have the same issue

**Comment by serkrush** [comment]
I have the same issue

**Comment by markusunger** [comment]
I have opened a PR to add this functionality: https://github.com/MobileReality/react-native-select-pro/pull/263
